package.json

```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "heroku-postbuild": "npm run build:frontend && npm run build:server",
     "start:frontend": "cd frontend && npm run start",
-    "start:server": "cd server && npm run start",
+    "start:server": "cd server && npm run dev",
```

Comment:
What's the reason for this change?

Reply:
for development, we would be using nodemon. However, there were issues with using nodemon in deployment

Reply:
Gotcha. `start` is generally reserved for execution in prod rather than dev, so it's probably better to introduce a new `dev:server` command if we wanted that.